src/test/java/api/subscribers/CloseDeclaredLostLoanWhenLostItemFeesAreClosedApiTests.java

```diff
@@ -0,0 +1,141 @@
+    assertThat(itemsClient.getById(item.getId()).getJson(), isDeclaredLost());
+  }
+
+  @Test
+  public void shouldNotCloseLoanIfItemFeeIsNotClosed() {
+    feeFineAccountFixture.payLostItemProcessingFee(loan.getId());
+
+    eventSubscribersFixture.publishLoanRelatedFeeFineClosedEvent(loan.getId());
+
+    assertThat(loansFixture.getLoanById(loan.getId()).getJson(), isOpen());
+    assertThat(itemsClient.getById(item.getId()).getJson(), isDeclaredLost());
+  }
+
+  @Test
+  public void shouldNotCloseLoanIfActualCostIsUsed() {
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .chargeProcessingFee(10.00)
+        .withActualCost(10.0)).getId());
+
+    feeFineAccountFixture.payLostItemProcessingFee(loan.getId());
+
+    eventSubscribersFixture.publishLoanRelatedFeeFineClosedEvent(loan.getId());
+
+    assertThat(loansFixture.getLoanById(loan.getId()).getJson(), isOpen());
+    assertThat(itemsClient.getById(item.getId()).getJson(), isDeclaredLost());
+  }
+
+  @Test
+  public void shouldNotCloseCheckedOutLoan() {
+    item = itemsFixture.basedUponNod();
+    loan = checkOutFixture.checkOutByBarcode(item, usersFixture.jessica());
+
+    eventSubscribersFixture.publishLoanRelatedFeeFineClosedEvent(loan.getId());
+
+    assertThat(loansFixture.getLoanById(loan.getId()).getJson(), isOpen());
+    assertThat(itemsClient.getById(item.getId()).getJson(), isCheckedOut());
+  }
+
+  @Test
+  public void shouldNotCloseLoanIfLoanIdDoesNotPresent() {
+    final Response response = eventSubscribersFixture
+      .attemptPublishLoanRelatedFeeFineClosedEvent(null, UUID.randomUUID());
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Loan id is required"),
+      hasNullParameter("loanId")
+    )));
+  }
+
+  @Test
+  public void shouldNotCloseNonExistentLoan() {
+    final UUID loanId = UUID.randomUUID();
+    final Response response = eventSubscribersFixture
+      .attemptPublishLoanRelatedFeeFineClosedEvent(loanId,
+        UUID.randomUUID());
+
+    assertThat(response.getStatusCode(), is(404));
+    assertThat(response.getBody(),
+      is(String.format("loan record with ID \"%s\" cannot be found", loanId)));
+  }
+
+  @Test
+  public void shouldNotCloseRefundedLoan() {
+    feeFineAccountFixture.payLostItemFee(loan.getId());
+    feeFineAccountFixture.payLostItemProcessingFee(loan.getId());
+
+    checkInFixture.checkInByBarcode(item);
+
+    assertThat(loansFixture.getLoanById(loan.getId()).getJson(), allOf(
+      isClosed(),
+      hasJsonPath("action", "checkedin")));
+    assertThat(itemsClient.getById(item.getId()).getJson(), isAvailable());
+
+    eventSubscribersFixture.publishLoanRelatedFeeFineClosedEvent(loan.getId());
+
+    assertThat(loansFixture.getLoanById(loan.getId()).getJson(), allOf(
+      isClosed(),
+      hasJsonPath("action", "checkedin")));
+    assertThat(itemsClient.getById(item.getId()).getJson(), isAvailable());
+  }
```

Comment:
Please could this be moved to be with the other should not close tests, rather than below the failure tests